src/pupil_labs/invisible_lsl_relay/outlets.py

```diff
@@ -91,11 +114,24 @@ def pi_streaminfo(
         channel_format=channel_format,
         source_id=outlet_uuid,
     )
-    stream_info.desc().append_child_value("pupil_invisible_lsl_relay_version", VERSION)
+    xml_acquisition = stream_info.desc().append_child("acquisition")
+    for key in acquisition_info.keys():
+        xml_acquisition.append_child_value(key, acquisition_info[key])
     xml_channels = stream_info.desc().append_child("channels")
     [chan.append_to(xml_channels) for chan in channels]
     return stream_info
 
 
 def get_lsl_time_offset():
     return time.time() - lsl.local_clock()
+
+
+def set_acquisition_info(
```

Comment:
This function does not really set anything but transform data.
```suggestion
def compose_acquisition_info(
```